### ISSUE TITLE
Revisit coins UI

### ISSUE THREAD

**Issue by tsusanka** [trezor/trezor-core#380]
The UI needs a complete revisit and unification of all coins. Each coin now does it a bit differently, which lowers the user experience.

Besides this, this issue also sets out to collect all minor tasks that we need to look into:

- [ ] Cardano uses pagination for every screen

**Comment by sergeylappo** [comment]
@tsusanka, @prusnak Trezor One had a cool feature. It was able to display that funds are going to Trezor's managed account. It was implemented https://github.com/trezor/trezor-mcu/pull/311.
Is there any plans to implement such a feature for Model T?
I even think that showing address in this case is redundant? or at least address is not the main thing to display.
I really like way it done in `one of your competitors` firmware (attached).
![image](https://user-images.githubusercontent.com/9092362/47678104-03e8be80-dbd2-11e8-8b3c-627f16d40bae.png), but for me it might be even better to show address path.
Something like:
``` 
  Transferring %f BTC to Segwit Account #%d.\n
  Derivation path: m/84'/1'/0'/0/50\n
```

**Comment by prusnak** [comment]
@sergeylappo yes, see checklist in the first comment - that's what I meant by "qualified paths", i.e. not showing `m/49'/1'/0'/0/4` but rather "Testnet Account #0 Address #4" or simply "Address from Testnet Account #0"

**Comment by sergeylappo** [comment]
@prusnak, but there's clearly stated that it's get address dialog, not signing process.

> GetAddress dialogs show paths, we should show coin name

**Comment by prusnak** [comment]
@sergeylappo correct, added new item to the list in the first comment

**Comment by refi93** [comment]
@tsusanka @prusnak any updates on the issue?

**Comment by prusnak** [comment]
@refi93 Not at the moment, we are still waiting for UX/UI designer to overhaul the UI on Trezor. This will take some time, though. Feel free to improve Cardano UX in the meantime (if you have particular ideas how to do that).

**Comment by tsusanka** [comment]
As stick mentioned, this waits for UX/UI designer. Moving to backlog.
